cmd/catalog-importer/cmd/app.go

```diff
@@ -30,6 +30,7 @@ var (
 	// Global flags
 	debug   = app.Flag("debug", "Enable debug logging").Default("false").Bool()
 	noColor = app.Flag("no-color", "Disable colored output").Default("false").Bool()
+	logJson = app.Flag("log-json", "Enable JSON logs").Default("false").Bool()
```

Comment:
I think it would be nice to have this as:

```suggestion
	logJson = app.Flag("log-format", "Format in which to emit logs (either json or logfmt)").Default("logfmt").String()
```